ee/codegen/src/generators/vellum-variable-value.ts

```diff
@@ -256,6 +257,47 @@ class AudioVellumValue extends AstNode {
   }
 }
 
+class FunctionCallVellumValue extends AstNode {
+  private value: FunctionCall;
+
+  public constructor(value: FunctionCall) {
+    super();
+    this.value = value;
```

Comment:
[blocking] We need to inherit references of the ast nodes that get generated.

Reply:
@noanflaherty Does this look right/is this what you meant? https://github.com/vellum-ai/vellum-python-sdks/pull/301/files#diff-669580c6ba7a67bf8abb45c1cf2dfeed737dcda7d72fb6eca8bc8bedded16742R289-R299

Reply:
We already inherit references here for the node that gets constructed at the end in vellum value:
https://github.com/vellum-ai/vellum-python-sdks/pull/301/files#diff-669580c6ba7a67bf8abb45c1cf2dfeed737dcda7d72fb6eca8bc8bedded16742R416

Reply:
@NgoHarrison Does this mean we don't need to inherit references on the more specific VellumValue types?